sigridci/sigridci/feedback_provider.py

```diff
@@ -55,6 +55,9 @@ def generateReports(self):
         if not os.path.exists(self.options.outputDir):
             os.mkdir(self.options.outputDir)
 
+        with open(f"{self.options.outputDir}/{self.capability.shortName}.json", mode="w", encoding="utf-8") as f:
```

Comment:
Consider making this more robust. Suppose a user configured as `outputDir` a directory that is shared by multiple CI/CD jobs/pipelines. Then this either errors, or overwrites results, which might be very confusing. The OS typically provides a way to create directories that are guaranteed to be unique, so you'd get something like this:
```suggestion
        subDir = os.createTempDir() 
        with open(f"{self.options.outputDir}/{subDir}/{self.capability.shortName}.json", mode="w", encoding="utf-8") as f:
            print(f"Output written to self.options.outputDir/{subDir}")
```

Reply:
Fine with me, but I guess we still need to print the filename. Although now that I think of it: ultimately, the idea is that users use the "artefacts" feature of their CI/CD environment. E.g., in GitLab, I'd probably just tell it that the entire `outputDirectory` needs to be saved as an artefact. Most of the time, it will not be shared, so it will have just one file for each capability, which is clear enough.